test/05-supervisor-release.ts

```diff
@@ -52,4 +52,110 @@ describe('supervisor release', function() {
 			.delete(`/resin/supervisor_release(${supervisorReleaseId})`)
 			.expect(200);
 	});
+
+	describe('Devices running supervisor releases', () => {
+		let applicationId: number;
+		let device: fakeDevice.Device;
+		let small: number;
+		let middle: number;
+		let large: number;
+		before(async () => {
+			// create a new test application...
+			const { body: application } = await supertest(app, admin)
+				.post('/resin/application')
+				.send({
+					device_type: 'intel-nuc',
+					app_name: 'supervisor_release_tes_app',
+				})
+				.expect(201);
+
+			applicationId = application.id;
+
+			device = await fakeDevice.provisionDevice(admin.token, applicationId);
+
+			// let's create some releases
+			const res = await supertest(app)
+				.get(`/resin/device_type?$select=id&$filter=slug eq 'intel-nuc'`)
+				.expect(200);
+
+			expect(res.body)
+				.to.have.nested.property('d[0].id')
+				.that.is.a('number');
+
+			middle = (
+				await supertest(app, admin)
+					.post(`/resin/supervisor_release`)
```

Comment:
These look like they should be in the fixtures

Reply:
Yup and I looked at adding them but it didn't look like the infrastructure is there yet?